Fix for resolving source when accessing sub-attr eg b.c

cosmosgenius opened the pull request:
```
plan = PlanSerializer(source='subscription.plan')
```
 `plan` is a field of another serializer which needs to be resolved using `obj.subscription.plan` property. It works for standard DRF JSON response but fails in drf-json-api.

This PR fixes that issue by replacing `getattr` with `operator.attrgetter` which is able to resolve  the sub-attr access.

cosmosgenius commented:
ping!. Is there anything else which needs to be done to merge this PR?

mblayman commented:
Hi, @cosmosgenius, thanks for your patience. Anyone working on this project is purely a volunteer so I guess we've all been fairly busy lately. Through no fault of your own, the tests are all failing. I believe this is because of a dependency problem that was resolved recently. Could you merge the latest `develop` into this branch? The project policy is to merge PRs only when they pass all tests. Once this branch is passing, I'm happy to merge as the fix content looks good. Thanks! 👍

cosmosgenius commented:
@mblayman no issues. Just checking up :). I have rebased the branch with the latest `develop` and fixed an isort issue. Let me know if anything else is required.

mblayman commented:
Awesome. Thanks, @cosmosgenius!
